src/DetailsView/components/save-assessment-factory.tsx

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as React from 'react';
+import { CommandBarProps } from 'DetailsView/components/details-view-command-bar';
+import { DetailsViewActionMessageCreator } from '../actions/details-view-action-message-creator';
+import { SaveAssessmentButton } from 'DetailsView/components/save-assessment-button';
+
+export type SaveAssessmentFactoryDeps = {
+    detailsViewActionMessageCreator: DetailsViewActionMessageCreator;
+};
+
+export type SaveAssessmentFactoryProps = CommandBarProps & {
+    deps: SaveAssessmentFactoryDeps;
```

Comment:
It looks like you aren't using these deps or props, so you'd be fine to take them out if you wanted. I know you'll probably need them later, though

Reply:
I think I'll leave it for now, since I'll need it for the next PR.